tests/treemodel/treewalker.js

```diff
@@ -16,11 +16,9 @@ import Range from '/ckeditor5/engine/treemodel/range.js';
 import CKEditorError from '/ckeditor5/utils/ckeditorerror.js';
 
 describe( 'TreeWalker', () => {
-	let expectedSingle, expected, expectedIgnoreEnd, expectedSingleIgnoreEnd;
-	let expectedShallow, expectedCroppedStart, expectedCroppedEnd;
-
+	let expected;
```

Comment:
I do not like this variable here. Each tests block use it and if one block forget to change the value of the `expected` it will use expected values from the previous block. Each block should define `expected` so if one forget about it, it will be undefined.